api-tests/pom.xml

```diff
@@ -141,8 +142,8 @@
         </dependency>
         <dependency>
             <groupId>org.hamcrest</groupId>
-            <artifactId>hamcrest-all</artifactId>
-            <version>${hamcrest.all.version}</version>
+            <artifactId>hamcrest</artifactId>
+            <version>${hamcrest.version}</version>
```

Comment:
Should be ${hamcrest.all.version} as defined on line 37. Or line 37 should be renamed